components/event/EventKey.js

```diff
@@ -17,6 +19,11 @@ export default function EventKey() {
             These are in person events
           </p>
         </div>
+        <div
+          className="flex items-baseline text-2xl font-semibold"
```

Comment:
Btw is `flex` needed here?

Reply:
Not really since this div is within a flex container already...just removed flex and items-baseline